src/Components/Main/App/Content/Users/Users.js

```diff
@@ -17,9 +17,9 @@ export default class Users extends React.Component {
 	}
 
 	getAllUserData() {
-		let response = this.props.getRequest("http://localhost:2000/users/");
-		console.log(response);
-		return response;
+		this.props.getRequest("http://localhost:2000/users/", (res) => {
```

Comment:
It's already correct. I've defined a function which takes URL, sends token along with get request and then returns response asynchronously, and it's named "getRequest". Callback will change depending on what component is firing it, so better just return response and deal with it later.
`getRequest` in some other file. Did it to reduce code duplication across files. 

Looks like this : 

`	getRequest(url) {
		var token = localStorage["aiims-login-token"]
			.replace(/^"/, "")
			.replace(/"$/, "");
		$.ajax({
			url: url,
			type: "GET",
			datatype: "application/json",
			data: { token: token }
		})
			.done(res => {
				return res;
			})
			.fail(err => {
				return err;
			});
	}`